docs/user-guide/api.md

```diff
@@ -52,11 +52,14 @@ For more information and examples, check out our [API documentation](https://api
 
 ## Authorization and Authentication
 
-For Authentication we're using [JSON Web Tokens]. They need to be passed via
-an `Authorization` header. To generate a JWT, visit the `/v4/auth/login` endpoint which will redirect you to the `/v4/auth/token` endpoint.
+For Authentication we're using [JSON Web Tokens] and Screwdriver API Tokens. JSON Web Tokens need to be passed via
```

Comment:
For authentication, we're using [JSON Web Tokens] (JWTs) and Screwdriver API Tokens. JWTs need to be passed via an `Authorization` header.